README.md

```diff
@@ -1,6 +1,35 @@
-# FRX Challenges
+# Frictionless Data Exchanges (FRX) for data challenges
 
-## The Helm chart
+Welcome to the FRX Challenges project! This is an open source repository that provides key software components for running competitive data science challenges.
+
+FRX stands for **F**rictionless **R**eproducibility e**X**change. Inspired by [Donoho, 2023](https://doi.org/10.48550/arXiv.2310.00865). This project enables communities to leverage cloud infrastructure and interactive computing tools to **host data challenges with live computation**.
+
+## Target functionality and goals
+
+This is a young project that is under active development.
+Below are the core workflows that we wish to support:
+
+- **prompts**: Allow organizers to create a website that describes the data challenge and provides instructions for participants.
+- **submissions**: Allow participants to create one or more submissions for evaluation.
+- **evaluators**: Leverage cloud infrastructure to run submissions against standardized environments and datasets, and allow organizers to define their own evaluation scripts, criteria, and metrics.
+- **feedback**: Provide information to participants about how their submissions scored relative to others.
+- **teams**: Allow participants to submit and view their results as a team of people.
+
+## About this project and acknowledgements
+
```

Comment:
```suggestion

We aim for this project to work **locally** as well as **via cloud infrastructure**, so that users may prototype and make submissions via their own hardware and environments, or via community-hosted infrastructure like **JupyterHub** and **BinderHub**.

## About this project and acknowledgements

```